pandas/tests/indexing/test_indexing.py

```diff
@@ -571,17 +570,23 @@ def test_astype_assignment_with_dups(self):
         # result = df.get_dtype_counts().sort_index()
         # expected = Series({'float64': 2, 'object': 1}).sort_index()
 
-    def test_coercion_with_contains(self):
+    @pytest.mark.parametrize("val,expected", [
+        (np.inf, False),
+        (np.nan, False),
+        (3.0, True),
+        (3.5, False),
+        (0.0, True),
+    ])
+    def test_coercion_with_contains(self, val, expected):
```

Comment:
Not quite what I was going for here.  What I actually was asking you to do was create example indices that *contain* `np.inf` or `np.nan`.  You can by all means keep the ones you have, but include test cases where testing membership of `np.inf` or `np.nan` returns true.